### PR TITLE
Support dark style

### LINKED ISSUE #2192: Support dark color schemes
With browsers across macOS, Android, Windows, and soon iOS and GNOME Web supporting the `prefers-color-scheme` media query, we should support it for our site as well. This involves a few tasks:

- [ ] Support the media query itself with a specific dark color scheme
- [ ] Check images and ensure major ones (i.e. on the front page) don't have hardcoded white backgrounds—instead, we should use transparent PNGs
- [ ] Test at least on macOS Safari and Chrome Canary for Android

### PR DESCRIPTION
**Pull request by cassidyjames**
Intended to fix #2192.

- Adds several palette colors to be used elsewhere
- Switches a couple of JPG references to PNGs

dark

- [ ] Button styles
- [x] Brand logomark backgrounds
- [x] Get Involved tweaks
- [x] Open Source chart and logo colors
- [ ] Store popups are white-on-white
- [ ] Language popover is always white

### CODE REVIEW

File: _styles/home.css [hunk 1]
```diff
@@ -81,11 +93,17 @@ input:focus + .focus-reveal {
...
+
+@media (prefers-color-scheme: dark) {
+    #the-press svg * {
+        fill: var(--silver-500);
+    }
 }
 
 #the-press svg:hover * {
-    fill: #3689e6;
+    fill: var(--bluebrry-500);
```
Comment (btkostner) [inline _styles/home.css]:
```suggestion
    fill: var(--blueberry-500);
```

**Comment by SeanRParker** [comment]
Is this still open? I’d love to take the remaining tasks on. Any color preferences for buttons? or are those color vars already declared?

**Comment by cassidyjames** [comment]
@SeanRParker yes, this would still be great to get wrapped up! The main issue with the button styles is that the strokes/shadows are meant for a white background, so they look weird on dark backgrounds. We can use transparency (like I did here: https://github.com/elementary/blog-template/pull/11) to work around that, or we could provide separate button styles for the dark style; whichever makes the most sense.

**Comment by SeanRParker** [comment]
I don't typically run php so set up took me a bit on my macbook. I have things up though and the button looks good. I'd start on the `#choice-buttons`, but I got to get to bed. I hope to look at those next.

Screen Shot 2019-10-19 at 10 40 42 PM

**Comment by SeanRParker** [comment]
Screen Shot 2019-10-22 at 10 37 54 PM
Screen Shot 2019-10-22 at 10 38 06 PM

Ok. I'm ready to push changes up but I think I need collaborator access? Should I push these to a different branch for a PR or is here fine?

**Comment by SeanRParker** [comment]
@cassidyjames ^

**Pull request closed without merging**